src/assets/docker-compose.yml

```
@@ -57,8 +59,9 @@ services:
       start_period: 10s
       test: ["CMD", "curl", "-f", "http://localhost:3000/api/health"]
       interval: 5s
-      timeout: 3s
+      timeout: 10s
       retries: 20
+      start_period: 5s
```

YAMLlint (1.35.1)

[error] duplication of key "start_period" in mapping

(key-duplicates)